src/actions/Find.ts

```diff
@@ -0,0 +1,26 @@
+import { ActionReturnValue, Graph, TypedSelection } from "../Types";
+import { commands } from "vscode";
+import GetText from "./GetText";
+
+class FindBase extends GetText {
```

Comment:
Why does this extend `GetText`? If you need to get text, I'd just do it directly, or if you'd like to use the action just call it from the graph. See eg `carve`

Reply:
I was thinking that we could have optional constructor arguments. So a subclass is better in that regard then calling via graph, but for now it doesn't really matter so calling via graph is fine if you prefer that.

Reply:
Sorry I'm not sure I follow. You're not passing any constructor arguments to the base class, and it doesn't have any optional arguments

Reply:
Not yet. For phones and such. Was a future provision. But for find it does t matter.